tests/dist/src/test/java/org/hawkular/apm/tests/dist/TraceCompletionTimeITest.java

```diff
@@ -339,17 +339,17 @@ public void testGetCompletionTimesThreeFragmentControlFlowMultiConsumerAsync() t
     protected void testGetCompletionTimesThreeFragmentMultiConsumerAsync(Scope scope, String suffix) throws Exception {
         Trace trace1 = new Trace();
         trace1.setId("1_"+suffix);
-        trace1.setStartTime(System.currentTimeMillis() - 60000); // Within last hour
+        trace1.setStartTime(TimeUnit.MILLISECONDS.toMicros(System.currentTimeMillis()) - 4000); // Within last hour
```

Comment:
Just from a review perspective, it makes it easier if there is a simple correlation between the pre/post change, as this is a large PR. Changing the values (other than from the perspective of units) means the review is more involved, to determine if the individual tests still make sense. But if you are sure they represent the same conditions, then it is not a major issue.